src/components/Data/Documents/Page.vue

```diff
@@ -390,6 +405,41 @@ export default {
     }
   },
   methods: {
+    async subscribeToCurrentDocs() {
+      try {
+        if (this.subscribeRoomId) {
+          await this.$kuzzle.realtime.unsubscribe(this.subscribeRoomId)
+          this.subscribeRoomId = null
+        }
+        const roomId = await this.$kuzzle.realtime.subscribe(
+          this.indexName,
+          this.collectionName,
+          {
+            or: this.documents.map(d => ({ equals: { _id: d._id } }))
+          },
+          this.realtimeNotifCallback
+        )
+        this.subscribeRoomId = roomId
+      } catch (error) {
+        this.$log.error(error)
+      }
+    },
+    realtimeNotifCallback(notif) {
+      if (notif.scope === 'out') {
+        return
+      }
+      if (!['replace', 'update'].includes(notif.action)) {
+        return
+      }
+      const documents = JSON.parse(JSON.stringify(this.documents))
```

Comment:
why is this?

Reply:
I'll update it using lodash 
For the explanation see comment below